WooCommerce/WooCommerceTests/ViewRelated/Survey/SurveyViewControllerTests.swift

```diff
@@ -32,8 +56,9 @@ final class SurveyViewControllerTests: XCTestCase {
         // Then
         XCTAssertTrue(mirror.webView.isLoading)
         XCTAssertEqual(mirror.webView.url, WooConstants.URLs.productsFeedback
-                        .asURL().tagPlatform("ios")
-                        .tagAppVersion(Bundle.main.bundleVersion()))
+                        .asURL()            .tagPlatform("ios")
```

Comment:
Super-nit: should there be a line break here?

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/15353/commits/882701ae07a5a6d488e15098b2503f89e3faa2f0